src/container/RelayRenderer.js

```diff
@@ -283,25 +293,22 @@ class RelayRenderer extends React.Component {
   }
 }
 
-function createFragmentPointerForRoot(query) {
-  return query ?
-    GraphQLFragmentPointer.createForRoot(
-      RelayStoreData.getDefaultInstance().getQueuedStore(),
-      query
-    ) :
-    null;
-}
-
 RelayRenderer.propTypes = {
   Component: RelayPropTypes.Container,
   forceFetch: PropTypes.bool,
   onReadyStateChange: PropTypes.func,
   queryConfig: RelayPropTypes.QueryConfig.isRequired,
   render: PropTypes.func,
+  relayContext: RelayPropTypes.RelayContext.isRequired,
+};
+
+RelayRenderer.defaultProps = {
+  relayContext: RelayStore,
```

Comment:
Default was added here because _tons_ of tests are shallow rendering a `RelayRenderer`. Will update those when I get a chance :+1: